AppleMusicClone/Views/SearchViews/SearchView.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  SearchView.swift
+//  AppleMusicClone
+//
+//  Created by Hankyu Lee on 2022/07/05.
+//
+
+import SwiftUI
+import Combine
+
+struct SearchView: View {
+    
+    @State private var location = ""
+
+    @StateObject var searchVM: SearchViewModel = SearchViewModel()
+    
+    var body: some View {
+        
+        VStack {
+            
+            TextField("Your Location", text: $location)
+                        .onChange(of: location) { _ in
+                            
+                            searchVM.fetch(location)
+                            
+                        }
+                        .padding()
+            ScrollView(.vertical) {
+                LazyVGrid(columns: [GridItem(.flexible())]) {
+                    LazyVStack {
+                        Text("Hints")
+                        ForEach(
+                            searchVM.hints3.indices
+                            , id: \.self) { index in
+                            Text(searchVM.hints3[index].inform)
+                        }
+                    }
+                    Rectangle()
+                        .frame(height:3)
+                        .padding()
```

Comment:
```suggestion
                    Divider()
```
할당받은 파트 외의 작업이지만 추후 Divider로 바뀌어야할것 같네요~